app/helpers/samples_helper.rb

```diff
@@ -88,7 +88,7 @@ def samples_output_data(samples)
     final_result = []
     samples.each do |output|
       output_data = {}
-      pipeline_output = output.pipeline_runs.first ? output.pipeline_runs.first.pipeline_output : nil
+      pipeline_output = output.pipeline_runs && output.pipeline_runs.first ? output.pipeline_runs.first.pipeline_output : nil
```

Comment:
i don't think you need to change this line. ouput.pipeline_runs will be [] if it's empty